aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -395,6 +395,7 @@ module supra_framework::pbo_delegation_pool {
     }
 
     struct AddStakeEvent has drop, store {
+        stake_funder: address,
```

Comment:
Better not to add this here so that we can preserve backwards compatibility with testnet.

Reply:
I think we can leave this information out of the event for now. As you said, coin transfer events will be emitted too, so we can use those to identify the funder if need be.

Reply:
fixed in added commits